classes.html

```diff
@@ -2,12 +2,29 @@
 {% block session_body %}
 
 <h3>Classes</h3>
-<table width='50%'><tr><td>
-TIP: Each class must have the following fields:
-name, max_enrollment, schedule-daypart, schedule-location.<br>
-Each class can also have: instructor, prerequisites, description, donation, exclude_from_catalog.<br>
-To exclude a class from the course catalog, set exclude_from_catalog: True. Otherwise, leave it out entirely. It is not necessary to set to False.
-</td></tr></table>
+<table width='50%'><tr><td style="color:red">
+CAUTION!!! CLASS ID'S WILL BE OVERWRITTEN!!!<br>
+</td></tr>
+<tr><td>
+Unless you understand the circumstances under which class ids are generated, please don't click submit!<br>
```

Comment:
Maybe we should check if any students already signed up for any classes. If they already have then make sure all those class_ids are still filled in and not auto assigned? How did this page mess you up before? Maybe lets chat about how it failed last time so we can design it to work better for you.